gitpod-io/gitpod
# [code] smooth VS Code desktop flow

#### What it does

fix https://github.com/gitpod-io/gitpod/issues/4780

Changes in Gitpod Code: https://github.com/gitpod-io/vscode/commit/b1465c2c89bdf60367db35943e96015faa9f89dd

#### How to test

- Go to preferences and enable latest VS Code: https://ak-vscode-desktop.staging.gitpod-dev.com/preferences
- Start a workspace: https://ak-vscode-desktop.staging.gitpod-dev.com/#github.com/akosyakov/parcel-demo
- File -> Open in VS Code
- First time it will ask you to install Gitpod extension, you will also go through the auth flow, and eventually see the remote ssh window for Gitpod workspace.
- Try to close and open again, second time the flow should be smoother.
